src/@chakra-ui/gatsby-plugin/components/Button.ts

```diff
@@ -30,7 +31,16 @@ const baseStyle = defineStyle((props) => ({
   _hover: {
     color: "primary.hover",
   },
-}))
+  "&[data-secondary='true']": {
+    color: "body.base",
+  },
+  "&.chakra-link": {
+    textDecoration: "none",
+    _hover: {
+      textDecoration: "none",
+    },
+  },
```

Comment:
For any instance `Button` is a link, typically consuming a component that uses Chakra's `Link`

Reply:
Nice simplification, this would cover also the cases where we apply a Link to an IconButton. Well...any instance as you said haha